src/main/java/com/kodilla/game/JavaFxMove.java

```
@@ -0,0 +1,9 @@
+package com.kodilla.game;
+
+public class JavaFxMove implements Move<String> {
+
+    @Override
+    public String getPosition() {
+        return null; //get Button id
```

better to throw UnsupportedOperationException, MethodNotImplementedYetException